src/components/AccountSwitchInput/AccountSwitchInput.tsx

```diff
@@ -27,11 +31,11 @@ const AccountSwitchInput = ({
   showTotals = false,
 }: AccountSwitchInputProps) => {
   let walletOptions = [
-    { title: 'Smart Wallet', value: DestinationWalletEnum.Contract},
+    { title: smartWalletTotal ? `${smartWalletTotal} Smart Wallet`: 'Smart Wallet', value: DestinationWalletEnum.Contract},
```

Comment:
Hey So I handled the total display through this ticket https://linear.app/pillarproject/issue/PRO-727